src/modules/resource/status.c

```diff
@@ -8,6 +8,12 @@
  * SPDX-License-Identifier: LGPL-3.0
  \************************************************************/
 
+/* status.c - handle resource.status and resource.sched-status RPCs
+ *
+ * Although this portion of the resource module is initialized on all
+ * ranks, the RPCs are denied if requested on rank > 0.
+ */
+
```

Comment:
Not critical, but it seems like this comment could go in a separate commit? (The comment does make me wonder why the status portion of the `resource` module is initialized on all ranks when RPCs are denied on those ranks other than 0, though)

Reply:
Sure, will do.

> The comment does make me wonder why the status portion of the resource module is initialized on all ranks when RPCs are denied on those ranks other than 0, though)

Just to deny the RPCs with a better message than "service not implemented' I think?  Hmm, maybe not worth it.  :shrug:

Reply:
Eh, not really worth addressing now I'd say